common/src/chain/config/checkpoints.rs

```diff
@@ -59,22 +93,78 @@ impl Checkpoints {
             .expect("Genesis must be there, at least.");
         (*cp_before.0, (*cp_before.1))
```

Comment:
we can remove the above exact match get() and just use range(..=height)

Reply:
Done